test/auth.test.ts

```diff
@@ -353,4 +353,46 @@ describe("Authentication", () => {
 
     expect(mock.done()).toBe(true);
   });
+
+  it("should pass through the logger", async () => {
```

Comment:
I like adding an issue number to tests so people can look up the decision process behind this functionality

```suggestion
  it("should pass through the logger (#1277)", async () => {
```

Reply:
Nice, I hadn't thought of doing that before